.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,64 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Has it been decided that all observability charms will use CodeQL ? (if not I would be inclined to remove it - change requested here)